backend.pm

```diff
@@ -589,8 +593,9 @@ my $ssh_params = [
 	['key_file', 'ssh_key', 'File for public key authentication'],
 	['serial_relay_port', 'serial_relay_port', "Serial relay poor man's reset dongle port"],
 	['reset_command', 'reset_command', 'If SUT hang, given command is '
-			. 'executed to reset. If command exit with error, test gets '
-			. 'stopped otherwise ssh connection will be reinitalized. ']
+	 . 'executed to reset. If command exit with error, test gets '
+	 . 'stopped otherwise ssh connection will be reinitalized. '],
```

Comment:
Why do you reformat these two lines?

Reply:
As @pevik said, please do not add unrelated changes to the patch. If you ever think that whitespaces should be formatted differently we can talk about that and do the changes in a separate patch.

Reply:
my bad. that was not intentional. it happens automatically on my editor when i added the comma and add a new line. i will remove